pkg/controller/conditions_test.go

```diff
@@ -37,7 +37,9 @@ var _ = Describe("VirtualMachineInstance ConditionManager", func() {
 	var pc2 *v12.PodCondition
 
 	BeforeEach(func() {
-		vmi = api.NewMinimalVMI("test")
+		vmi = libvmi.New(
+			libvmi.WithName("test"),
+		)
```

Comment:
`libvmi.New` will generate a random name. Can you please remove the `WithName` option?

```suggestion
		vmi = libvmi.New()
```